contrib/win32/openssh/install-sshd.ps1

```diff
@@ -84,6 +85,26 @@ if (Test-Path $moduliPath -PathType Leaf)
     Repair-ModuliFilePermission -FilePath $moduliPath @psBoundParameters -confirm:$false
 }
 
+#If %programData%/ssh folder already exists, fix permissions
+$sshProgDataPath = Join-Path $env:ProgramData "ssh"
```

Comment:
We should modify this logic just like contrib\win32\openssh\FixUserFilePermissions.ps1.
It first checks the file permissions. If the file permissions are violating then prompt user to fix (Y/N).
We should let users about the discrepancy and take their consent to fix the ACLs.